pkg/cluster/config/v1alpha2/conversion.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by conversion-gen. DO NOT EDIT.
+
+package v1alpha2
+
+import (
+	conversion "k8s.io/apimachinery/pkg/conversion"
+	config "sigs.k8s.io/kind/pkg/cluster/config"
+)
+
+func Convert_v1alpha2_Config_To_config_Cluster(in *Config, out *config.Cluster, s conversion.Scope) error {
+	// TODO(bentheelder): try to convert kubeadm config patches?
```

Comment:
we should definitely convert them going in this direction
/hold

cc @munnerz , this file is the hairy part. Technically we don't do round-tripping 1:1, but that's probably OK since we don't actually ever convert back to an external type and the types are alpha.